Cargo.toml

```diff
@@ -61,11 +62,13 @@ libloading = "0.7"
 
 [features]
 # By default libstd support and gimli-symbolize is used to symbolize addresses.
-default = ["std"]
+default = ["std", "rustc-demangle"]
 
 # Include std support. This enables types like `Backtrace`.
 std = []
 
+cpp_demangle = ["cpp_demangle_crate", "rustc-demangle"]
```

Comment:
It doesn't seem this actually depends on rustc-demangle.

Reply:
It doesn't, but without further changes to the symbolication logic it'll never try to demangle as C++ because the Rust demangling "succeeds". I could have also had the Rust demangling always *fail*, at the cost of losing the UTF-8 validation that had already been done. That would probably be more pedantically correct, but with such a narrow use case (demangle C++ but not Rust symbols) that the perf gain might be more worth it.